tests/unit/test_completer.py

```diff
@@ -77,16 +77,16 @@
     ('aws s3 m', -1, set(['mv', 'mb'])),
     ('aws s3 cp -', -1, set(['--no-guess-mime-type', '--dryrun',
                              '--recursive', '--website-redirect',
-                             '--quiet', '--acl', '--storage-class',
-                             '--sse', '--exclude', '--include',
-                             '--follow-symlinks', '--no-follow-symlinks',
-                             '--cache-control', '--content-type',
-                             '--content-disposition', '--source-region',
-                             '--content-encoding', '--content-language',
-                             '--expires', '--grants', '--only-show-errors',
-                             '--expected-size', '--page-size',
-                             '--metadata-directive',
-                             '--ignore-glacier-warnings']
+                             '--quiet', '--acl', '--copy-acl', '--no-copy-acl',
+                             '--storage-class', '--sse', '--exclude',
+                             '--include', '--follow-symlinks',
+                             '--no-follow-symlinks', '--cache-control',
+                             '--content-type', '--content-disposition',
+                             '--source-region', '--content-encoding',
+                             '--content-language', '--expires', '--grants',
+                             '--only-show-errors', '--expected-size',
+                             '--page-size', '--metadata-directive',
+							 '--ignore-glacier-warnings']
```

Comment:
Oops, sorry that last line did end up with tabs somehow (I resolved a merge conflict and I think the merge tool may have been configured for tabs). Everywhere else is spaces.

I'll fix this, and add the new options to the end as you suggest.